cfgov/jobmanager/tests/models/test_blocks.py

```diff
@@ -10,13 +11,15 @@
 from jobmanager.models.panels import GradePanel, RegionPanel
 
 
-def make_job_listing_page(title, close_date, grades=[], regions=[]):
+def make_job_listing_page(title, close_date=None, grades=[], regions=[],
+                          **kwargs):
     page = mommy.prepare(
         JobListingPage,
         title=title,
-        close_date=close_date,
+        close_date=close_date or timezone.now().date(),
```

Comment:
@vccabral It's `close_date` if it's not `None`, otherwise it's the value of `timezone.now().date()`.  It would only be assigned a boolean if one of those variables returned a boolean.